scripts/app.js

```diff
@@ -71,10 +71,46 @@ function handleLogin(event) {
 
 //add event listener to login
 var loginForm = document.getElementById('login');
+loginForm.addEventListener('submit', handleLogin);
 
-var x = new Addbeer('corona', '5%', 'lager', 'I mean its beer with lime usually', '2.5');
-console.log(x);
-console.log(Addbeer.beerDrink);
+//var x = new Addbeer('corona', '5%', 'lager', 'I mean its beer with lime usually', '2.5');
+//console.log(x);
+//console.log(Addbeer.beerDrink);
 
+var drinkHeader = document.getElementById('table-head');
+var drinkBody = document.getElementById('table-body');
+
+var createHeader = function () {
+  for (var x = 0; x < headerArray.length; x++) {
+    var categories = document.createElement('th');
+    categories.textContent = headerArray[x];
+    drinkHeader.appendChild(categories);
+  }
+};
+//Need to create seperate tr function;
+var tableRow = document.createElement('tr');
+
+var makeRow = function(){
+  tableRow = document.createElement('tr');
+  drinkBody.appendChild(tableRow);
+};
+
+var createDrinkData = function () {
+
+  for (var i = 0; i < headerArray.length; i++) {
+    makeRow();
+    var userDrinkData = document.createElement('td');
+
+    userDrinkData.textContent = Addbeer.beerDrink[i].name + Addbeer.beerDrink[i].abv + Addbeer.beerDrink[i].type + Addbeer.beerDrink[i].writtenNotes + Addbeer.beerDrink[i].score;
```

Comment:
If i'm reading this correctly, you have a TR at this point and then you make a TD.
The content of the TD is a concatenated string that's roughly "Budlight2.3LagerI think this beer is shallow and pedantic3"
When you look at the DOM is this creating just 1 TD for the row or is it creating more?